cpovc_forms/views.py

```diff
@@ -9749,57 +9749,90 @@ def new_hivmanagementform(request, id):
-                                                                                        'FirstLine_Start_Date',
-                                                                                        'Switch_SecondLine_ARV',
-                                                                                        'Switch_SecondLine_Date',
-                                                                                        'Switch_ThirdLine_ARV',
-                                                                                        'Switch_ThirdLine_Date')[0:1]
+            # ovc_hiv_obj = OVCHIVManagement.objects.filter(person=init_data).values_list('Hiv_Confirmed_Date',
+            #                                                                             'Treatment_initiated_Date',
+            #                                                                             'Switch_ThirdLine_Date')[0:1]
+            ovc_hiv_obj = OVCHIVManagement.objects.filter(person=init_data).all()[0:1]
+
             form_arvtherapy = HIV_MANAGEMENT_ARV_THERAPY_FORM(initial={'person': id})
+            form = HIV_MANAGEMENT_VISITATION_FORM(initial={'person': id})
             if(ovc_hiv_obj):
-                hiv_confirmed_date = 0
-                treatment_initiated_date = 0
-                firstLine_start_date = 0
-                substitution_firstLine_arv = 0
-                substitution_firstLine_fate = 0
-                switch_thirdLine_arv = 0
-                switch_thirdLine_date = 0
-
+                print "data available"
                 for hiv_obj in ovc_hiv_obj:
-                    hiv_confirmed_date = hiv_obj[0]
-                    treatment_initiated_date = hiv_obj[1]
-                    firstLine_start_date = hiv_obj[2]
-                    substitution_firstLine_arv = hiv_obj[3]
-                    substitution_firstLine_fate = hiv_obj[4]
-                    switch_thirdLine_arv = hiv_obj[5]
-                    switch_thirdLine_date = hiv_obj[6]
-
-                form_arvtherapy = HIV_MANAGEMENT_ARV_THERAPY_FORM(initial={'person': id,
+
+                    hiv_confirmed_date = hiv_obj.Hiv_Confirmed_Date
+                    treatment_initiated_date = hiv_obj.Treatment_initiated_Date
+                    firstLine_start_date = hiv_obj.FirstLine_Start_Date
+                    substitution_firstLine_arv = hiv_obj.Substitution_FirstLine_ARV
+                    substitution_firstLine_fate = hiv_obj.Substitution_FirstLine_Date
+                    #
+                    #
+                    switch_thirdLine_arv = hiv_obj.Switch_ThirdLine_ARV
+                    switch_thirdLine_date = hiv_obj.Switch_ThirdLine_Date
+                    Visit_Date = hiv_obj.Visit_Date
+                    Duration_ART = hiv_obj.Duration_ART
+                    #
+                    MUAC = hiv_obj.MUAC
+                    Adherence = hiv_obj.Adherence
+                    Adherence_Drugs_Duration = hiv_obj.Adherence_Drugs_Duration
```

Comment:
follow python conventions (https://www.python.org/dev/peps/pep-0008/) for creating variables. Please refer to link